### ISSUE TITLE
Unable to change profile with analog stick rebinding to keyboard

### ISSUE THREAD

**Issue by Limyx826** [Ryochan7/DS4Windows#281]
I currently using DS4Windows version 1.4.118. When trying to change to a profile with mapped keys on keyboard with force it to change to a profile bind to Xbox360 controller. One thing I notice is that a profile with the controller partially mapped to the keyboard works just fine, notably the analog stick is not mapped. When force to change to the problem profile, keys will repeating non stop. The issue that I found is that when you map a key on sticks the issue occurs.

**Comment by Limyx826** [comment]
Hello is anyone there? Anyway the issue persisted in version 1.4.119, I just hope that anyone can fix this issue.

**Comment by Ryochan7** [comment]
I will have to test it out but I am betting this problem affects the old Jays2Kings build as well. There is a very loose association between controller inputs and output actions particularly regarding emulated keyboard support. There is likely no release action being performed during the profile switch.

Also, can you share the profiles that you are using when you experience the issue? You might also have to share your Actions.xml file.

**Comment by Limyx826** [comment]
@Ryochan7 
I currently using both Jays2Kings lastest version along side with your lasted version. Jays2Kings version doesn't have this problem when I bind keyboard buttons on the analog stick. As for your version it can change between profiles with analog stick that bind to x360 sticks or didn't bind to the keyboard. When I try to change to the problem profiles it will randomly jump to a profile that doesn't has the binding. 

Here is the xml files you ask for.
[DS4Windows.zip](https://github.com/Ryochan7/DS4Windows/files/2065761/DS4Windows.zip)

**Comment by Ryochan7** [comment]
Theory. There is no dead zone for the analog stick in your profiles. DS4Windows used to have an implicit dead zone if the setting was 0 but that has not been the case for a while. When the profile is initially switched, a virtual key mapped to the analog stick is being activated. If the main Controllers tab is in view and a profile combobox has input focus, DS4Windows will switch to a profile that has that character as the first letter of the profile name.

I was going to change DS4Windows to ignore key presses for the combo boxes but the initial change made input performance dip. I ended up experimenting and changing other parts of the code and I did not go back to trying that change again.

My suggestion would be to enable a dead zone for LS in any profile that uses keyboard key bindings.
